test/unit/gateways/airwallex_test.rb

```diff
@@ -0,0 +1,200 @@
+require 'test_helper'
+
+module ActiveMerchant #:nodoc:
+  module Billing #:nodoc:
+    class AirwallexGateway
+      def setup_access_token
```

Comment:
Shadowing the method here so we don't have to make remote calls to do unit tests.